rclcpp/src/rclcpp/node.cpp

```diff
@@ -137,7 +137,8 @@ Node::Node(
       node_services_,
       node_logging_,
       node_clock_,
-      node_parameters_
+      node_parameters_,
+      options.start_parameter_event_subscriber()
```

Comment:
So this option is actually only for time_source? I was expecting that this works for all user operations as long as related to node.